Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -284,8 +288,8 @@
 - type: entity
   parent: ClothingOuterHardsuitBaseNFNfsd
   id: ClothingOuterHardsuitNfsdCommand
-  name: NFSD command hardsuit
-  description: Medium armored hardsuit helmet for NFSD raids.
+  name: NFSD command combat hardsuit
+  description: A more heavily armored NFSD suit. Still space proof, but with much better armor plating, at the expense of mobility. This variant has supervisor rank markings.
```

```suggestion
  description: A heavily armored NFSD suit. Space proof, with thick armor plating at the expense of mobility. This variant has supervisor rank markings.
```
No "more", more descriptive than "much better".